include/stronk/unit_v2.h

```diff
@@ -195,11 +206,13 @@ template<typename T, unit_value_like B>
 constexpr auto operator/(const T& a, const B& b) noexcept
 {
     auto res = a / b.template unwrap<B>();
-    using new_dimensions = typename B::unit_t::dimensions_t::negate_t;
-    using underlying_type = decltype(res);
-    using new_unit = typename unit_lookup<new_dimensions>::unit_t;
-    using new_unit_value = new_unit::template value<underlying_type>;
-    return new_unit_value {res};
+    using dimensions_t = typename B::unit_t::dimensions_t::negate_t;
+    using resulting_unit = typename unit_lookup<dimensions_t>::unit_t;
+    using underlying_t = decltype(res);
+    using scale_t = typename B::scale_t;
```

Comment:
Did not read all of it yet, but here we divide a non-unit by a unit, right?

So `1 / hour` (or `1 / (3600 seconds)`). Should the scale now still be 3600 seconds (per hour)?

Reply:
👍  Good catch!